### DISCUSSION TITLE
scrum prep Milestone 3 - 2

### DISCUSSION THREAD

**Discussion by Yahid1** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#156] [General]
Write your scrums here

**Comment by Janiel777** [comment]
Esta semana estuve buscando ideas para hacer LTT.

**Comment by Yahid1** [comment]
Determined some Objective tasks for the Metrics team.

**Comment by juan-ortiz92** [comment]
This week I finished my third lecture topic task and have been investigating to work on my last lecture topic task.

**Comment by Alej24** [comment]
Have been looking for lecture topic task ideas.

**Comment by dAninAd1248** [comment]
Had a meeting with the metrics team and planned for the next two sprints.

**Comment by AlexisTraversoPR** [comment]
Looking at LTT. Besides that not much.

**Comment by marcosarraiza** [comment]
Had a meeting with the metrics team and discussed what should be done for the next sprints.

**Comment by JoelTorres20** [comment]
Spent some time looking at lectures to work on my next LTT and this weekend I'll start on my current task.

**Comment by sebasagir** [comment]
looked for LTT, thought about code tasks and reviewed the milestone 2 retrospective

**Comment by Brandell13** [comment]
I was looking for what task I could work on this spring

**Comment by JaseyAcevedo** [comment]
Almost finished the task i was working on and looking for ltt

**Comment by yamayrie** [comment]
been reading class presentations to create an LLT

**Comment by JConcepcion01** [comment]
Didnt do much this week with the whole election things and holidays going on. But I'll start this weekend with my tasks and creating more.

**Comment by IvanierB** [comment]
Looking for 3 LTT to do for this milestone, besides that I have not been able to do much. I just came back from an academic trip and am now sick.

**Comment by MrL9000** [comment]
I started to set up my tasks for the milestone 3, then in the upcomming weekend I started working on them

My mistake for writing this scrum prep so late, I will try my best to not be late next time

**Comment by dylanDeJesus21** [comment]
This week worked on documentation
